CHANGELOG.md

```diff
@@ -28,6 +35,7 @@ All notable changes to the Zowe Client Python SDK will be documented in this fil
 
 ### Bug Fixes
 
+- Validate existing type annotations. [#321] (https://github.com/zowe/zowe-client-python-sdk/issues/321)
```

Comment:
Similar to previous comment - this bug fix should be moved under a `## Recent Changes` header at the top of the CHANGELOG.md file.